BNNApproximation.py

```diff
@@ -9,24 +9,24 @@ def __init__(self, state_dimensions, num_acts):
         super().__init__(state_dimensions, num_acts)
         self.session = tf.InteractiveSession()
         self.bnn = new_bnn()
-        #TODO refactor this back out into the BNN method
-        self.bnn.compile(optimizer=tf.keras.optimizers.Adam(1e-2),
+        self.bnn.compile(optimizer=tf.keras.optimizers.SGD(1e-1),
             loss=tf.losses.huber_loss)
-        # input = np.array(state_dimensions + self.convert_action(self.num_actions))
-        # self.history = self.bnn.fit(input, 0, batch_size=32,
-        #     epochs=1000, verbose=1)
 
 
-    def update_stats(self, x, val=0.0, batch_size = 32, epochs = 100):
+    def update_stats(self, x, val=0.0, batch_size = 1, epochs = 10):
```

Comment:
We _may_ even want to lower the number of epochs down to `10`. This parameter would affect how quickly our uncertainty dissipates after receiving a new observation.

Reply:
Do you mean you want to lower it even lower than  10? Agreed that we should do a parameter sweep on this as well